hack/build-go.sh

```diff
@@ -32,7 +32,7 @@ fi
 
 mkdir -p ${BIN_PATH}
 
-CGO_ENABLED=${CGO_ENABLED:-1}
+CGO_ENABLED=${CGO_ENABLED:-0}
```

Comment:
Why do we want static linkage?

Reply:
because now the sriov-config-daemon binary runs on the host and I don't want any gcc lib requirements

Reply:
do we need it only for config daemon or for operator too?

Reply:
it's possible to add this only for the config-daemon but what is the cons of having this to all the binaries?

Reply:
not a fan of static linking even though it's easier.